ios/engine/KMEI/KeymanEngine/resources/Keyman.bundle/Contents/Resources/ios-host.js

```diff
@@ -88,7 +86,7 @@ function setBannerImage(path) {
     }
 
     // If an inactive banner is set, update its image.
-    bc.inactiveBanner = bc.inactiveBanner ? new bc.ImageBanner(bannerImgPath) : null;
+    bc.inactiveBanner = new bc.ImageBanner(bannerImgPath);
```

Comment:
what is meant by an inactiveBanner? is that one that doesn't do predictive text? that's my best guess -- or maybe it doesn't wave in the breeze on windy days

Reply:
> is that one that doesn't do predictive text?

Bingo.  "Banner to use when predictive-text is inactive".